packages/core/src/lib/sign-in-experience.ts

```diff
@@ -56,7 +82,7 @@ export const validateSignInMethods = (
       'sign_in_experiences.enabled_connector_not_found',
       { type: ConnectorType.Social }
     );
-    // TODO: assertNonemptySocialConnectorIds
-    // TODO: assertEnabledSocialConnectorIds
+    assertNonemptySocialConnectorIds(socialSignInConnectorIds);
+    assertEnabledSocialConnectorIds(socialSignInConnectorIds, enabledConnectorInstances);
```

Comment:
nit: not necessarily extract to a separate method.

Reply:
I'm good with both.

- If inlined, this function will be a little longer: 44 lines.
- Even though it won't be reused in another place, I think it's more readable to extract it and give it a good name.

Reply:
Inlined.